.github/workflows/ci.yml

```diff
@@ -155,3 +155,24 @@ jobs:
 
     - name: cargo test --features nightly
       run: cargo test --features nightly
+
+  conformance-interface:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Cache cargo folder
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo
+          key: readme-doctest
```

Comment:
should have a different key, the readme-doctest are nightly artifacts which won't do anything if you're working with stable